guvnor-project/guvnor-project-api/src/main/java/org/guvnor/common/services/project/preferences/GAVPreferences.java

```diff
@@ -30,10 +30,12 @@
     static final String CHILD_GAV_EDIT_ENABLED = "org.guvnor.project.gav.child.edit.enabled";
 
     @Property(bundleKey = "GAVPreferences.ConflictingGAVCheckDisabled.Label",
+            helpBundleKey = "GAVPreferences.ConflictingGAVCheckDisabled.Help",
```

Comment:
@manstis Actually I formatted the file, unfortunately this is how annotation members get formatted...